auth0/src/test/java/com/auth0/android/provider/WebAuthProviderTest.java

```diff
@@ -1893,6 +1893,68 @@ public void shouldResumeLoginWhenJWKSRequestSuceeds() throws Exception {
         mockAPI.shutdown();
     }
 
+    @SuppressWarnings("deprecation")
+    @Test
+    public void shouldResumeLoginIgnoringEmptyCustomIDTokenVerificationIssuer() throws Exception {
+        WebAuthProvider.init(account)
+                .withResponseType(ResponseType.ID_TOKEN)
+                .withIdTokenVerificationIssuer(null)
```

Comment:
ensures that using "null" doesn't change the expected issuer to something different than the default value: the auth0 domain